pkgs/development/python-modules/os-service-types/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchurl
+, pbr
+, six
+
+, reno
+, tox
+}:
+let
+  pbr_5_5_0 = pbr.overrideAttrs (old: {
+    version = "5.5.0";
+    name = "pbr-5.5.0";
+    src = fetchPypi {
+      pname = "pbr";
+      version = "5.5.0";
+      sha256 = "1sl2w7zq33kzbh6im3lyl7jfwyddjkqmrx0y5b93v2n7a67xkgql";
+    };
+  });
+in
```

Comment:
having different versions of a python module can subtley break other packages at runtime.

This is only allowed with very few exceptions. Please try to remove it (relax version bounds on the package which is doing the pinning"